### ISSUE TITLE
Project help and support.

### ISSUE THREAD

**Issue by mchomem** [sinshu/managed-doom#1]
**Hello friend**

I did see your work and put a comment in your youtube video, congratulations for this.

**Questions:**

1. The sound don't work. Exists some initial setup on source code to do?
2. It's possible to provide anything help or contribution in your project? (despite having little time)
I was interested and would like to help with something.

OBS: A suggestion that can be included in the tasks is to move the character using the mouse.

**My PC setup:**

SO: Windows 10 Pro V 1909 - compilation 18363.778
Platformt: Intel(R) Core(TM) i7-2600K CPU @ 3,4 GHz
RAM: 8 GB
System type: 64 bits
Off-board video card: NVIDIA model GTX 1650 (with latest drive version)
On-board sound card: Realtek

**Used tools:**

Visual Studio 2019 community

**Comment by sinshu** [comment]
Thank you for your support.

Currently the audio system does not work due to some design changes.
I believe that most of the audio related code is still usable and the game sound will come back soon.

Your suggestion regarding the mouse support is nice.
I would also like to confirm how much CPU load is required to run this port on your PC.

**Comment by mchomem** [comment]
Follow a print screen to showing how much CPU load is required for your port:

![image](https://user-images.githubusercontent.com/6800486/80868511-8d6ac800-8c71-11ea-8139-af86dd21d424.png)

I did see little things about the source code, some classes.
You use interesting resources of c#.net that resemble something like C or C ++, cool.

I have little experience em C or C++ but believe that little by little and working on small resources, I can give considerable help.

When you can, you can delegate a small task to me so that I can work and start to help with code effectively.

It is better to do something targeted.

**Comment by mchomem** [comment]
Hello @sinshu .

Can you add me to your project?
And give me a task, please.

Good day for you.

**Comment by sinshu** [comment]
I considered how we can share the work, and found that it is getting difficult to find a task which can be easily delegated to others. Since this project started a number of months ago, most of the simple tasks are already done and the remaining tasks are complicated in some way.

Even one of the seemingly simplest tasks, refactoring the code from C style to C# style, is not that simple. Because this port aims to preserve the demo playback compatibility to the original version, every single change in the code must be tested to ensure that the change does not break the compatibility. To accomplish this, automated compatibility tests are included in this project. If you were to change the code, you would have to understand how the tests work and how to add new tests.

As explained above, now I realized that it is difficult to find a task for you.
I appreciate your willingness to help, and hope for your understanding.

**Comment by mchomem** [comment]
I did see a part of code, really the entire project is complex.
I need to be honest with you, I never did work with of game project, and searching by web, I found your project (I played this game the long ago), was because of this that was stay excited.
Although I do not be suitable for this project, I would like to want to accept the challenge.

But I understand your positioning.

@sinshu have a nice day!
